src/components/forms/charcount.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../colors";
+
+p {
```

Comment:
It looks like everything is in `.char-count`, I just put it on the next line in `&.char-count` because the linter was requesting it. Or should there not even be the `p` in here at all @rschamp perhaps?